latex2mmlc/src/parse.rs

```diff
@@ -240,8 +242,8 @@ where
                 // so that we can render them as percentages.
                 let lt = match self.parse_text_group()?.trim() {
                     "" => None,
-                    "0pt" => Some('0'),
-                    _ => return Err(LatexError(0, LatexErrKind::UnexpectedEOF)),
+                    decimal => Some(Length::from_str(decimal)
+                        .map_err(|LengthParseError| LatexError(0, LatexErrKind::UnexpectedEOF))?),
```

Comment:
There should be a dedicated error for this. The "UnexpectedEOF" here was just a placeholder. You can add the error in `latex2mmlc/src/error.rs`. This would then also close #57